lib/import-export-visitor.js

```diff
@@ -666,17 +713,19 @@ function safeParam(param, locals) {
 }
 
 function toModuleExport(specifierMap) {
-  const exportedKeys = specifierMap && Object.keys(specifierMap);
+  const exportedKeys = specifierMap ? Object.keys(specifierMap) : [];
```

Comment:
Here's a bit for reference https://github.com/lodash/lodash/commit/861aaf06c62a84e9c3ba952d9fa8d2d531ac3643. I can get away with `null` or `undefined` here if an empty array is too gross.